gridfs/__init__.py

```diff
@@ -664,10 +681,10 @@ def upload_from_stream_with_id(self, file_id, filename, source,
         """
         with self.open_upload_stream_with_id(
                 file_id, filename, chunk_size_bytes, metadata,
-                session=session) as gin:
+                session=session) as gin:  # type: GridIn
```

Comment:
Why is `# type: GridIn` needed here?

Reply:
Not needed anymore, must have been before full typing was added